### DISCUSSION TITLE
"User type can't be blank" when trying to apply for educational benefits.

### DISCUSSION THREAD

**Discussion by jbb42** [community/community#88611] [GitHub Education]
### Hi everyone,

When I try to apply for educational benefits, I get the following error:
"User type can't be blank
We require applicants to use a school-issued email address to apply."
Even though I have selected student as user type, and GitHub recognises my email and automatically finds my university. I have tried with different browsers and devices, but the error persists no matter what I do.

**Comment by solenn-tl** [comment]
Hi!
I have the same error than you (and try the same thing, without sucess)

**Reply by hosniomar01** [reply]
I have the same error too

**Comment by ghost** [comment]
Same problem

**Comment by ZoeLeBlanc** [comment]
Can confirm that students in my course are having this issue as well 😢

**Comment by ghost** [comment]
It must be a website error because it seems like it's not detecting the checkbox. It's strange.

**Reply by solenn-tl** [reply]
Yes, it seems to be something like that

**Comment by brianhill** [comment]
Is the answer going to be "have to use Chrome or Edge?" I am on Safari. Would rather not install other browsers.

**Comment by ghost** [comment]
I have analyzed the website and I believe the error is due to a problem in the JQUERY library. It's going to be a matter of waiting for the admins to fix this error or update the library

**Comment by Visuwanaath** [comment]
Same problem for me

**Comment by lguisadom** [comment]
same problem: Chrome. Ubuntu OS

**Comment by smayordomo** [comment]
Same problem with Firefox and Chrome. Ubuntu OS

**Comment by mine-tech-oficial** [comment]
Same problem with Firefox. Windows

**Comment by brianhill** [comment]
I was able to initiate a support request via another means. Yesterday, they responded to that request and said I would be able to proceed, and indeed, I was able. Is this fixed for everybody now?

**Comment by mine-tech-oficial** [comment]
Yes it worked, thx!

**Comment by zmrocze** [comment]
same problem

**Reply by Quart22** [reply]
I also cannot apply due to this error. How to fix it?

**Reply by dgorbunov** [reply]
I have the same issue

**Reply by fernandoxconceicao** [reply]
i too

**Reply by crnicholson** [reply]
Same problem (MacOS, Chrome, Brave, Safari)

**Reply by mpkendall** [reply]
same problem here

**Reply by RedMincrafter** [reply]
Same problem here (windows, chrome)

**Reply by PavanSETTEM-003** [reply]
same problem for me as well

**Comment by GeorgeRaducanu** [comment]
Same problem

**Comment by amirave** [comment]
same problem

**Comment by fernandoxconceicao** [comment]
Same problem

**Comment by dotKevinWong** [comment]
Seems like the site is completely broken. Looked in the console and there seems to be some bad JS.

I can't get past the onboarding page (https://education.github.com/onboardings/new) - the buttons don't do anything

I can get to the Education Discounts page (https://education.github.com/discount_requests/application) - however, I get the same error as everyone else on all browsers (Safari, Chrome, Firefox) on macOS where it says "User type can't be blank We require applicants to use a school-issued email address to apply."

Looking in the network tab, seems that the payload isn't picking up user type.

**Reply by crnicholson** [reply]
Someone should make a new issue

**Reply by dotKevinWong** [reply]
I re-verified my status about 10 minutes ago. Onboarding and Education Discount both work. GitHub fixed the issue. The console doesn't show the error previously seen.

**Comment by Q1CHENL** [comment]
same problem

**Comment by santiagosotoutn** [comment]
same problem

**Comment by Myoof** [comment]
same problem

**Comment by hilorioze** [comment]
same problem, firefox windows/android

**Comment by asgharmughal** [comment]
Students in my course facing same issue.

**Comment by mpkendall** [comment]
Works for me now, try again...
